src/tokens/newtabFontTokens.stories.svelte

```diff
@@ -1,8 +1,18 @@
 <script lang="ts">
+  import { onMount } from 'svelte'
   import { Meta, Story } from '@storybook/addon-svelte-csf'
   import { font as allFonts } from '../../tokens/css/variables-newtab'
   import FontTokenSwatchGroup from '../storyHelpers/FontTokenSwatchGroup.svelte'
-  import '../../tokens/css/variables-newtab.css'
+  // @ts-ignore
+  import styles from '../../tokens/css/variables-newtab.css?raw'
+
+  onMount(() => {
```

Comment:
Bit annoying that we have to do this separately for everything :/

Reply:
Had to tweak your suggested function since `onMount` can't be async AND return a function for unmount, but here's what it looks like:

https://github.com/brave/leo/pull/698/commits/871182b96465413e6fa694a2c620a92114f72b49

Pretty minor, but one down side here is that the css files get fetched for each story. Benefit probably outweighs the negative here, though.